hs_core/views/resource_folder_hierarchy.py

```diff
@@ -71,6 +69,14 @@ def data_store_structure(request):
     res_coll = os.path.join(resource.root_path, store_path)
     try:
         store = istorage.listdir(res_coll)
+        dirs = []
+        for dname in store[0]:  # directories
+            d_pk = dname.decode('utf-8')
+            name_with_full_path = os.path.join(res_coll, d_pk)
+            istorage = resource.get_irods_storage()
```

Comment:
@sblack-usu istorage is already defined above and used on line 71, so line 76 above is not needed.

Reply:
removing it.